blueprints/helpers.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const stringUtils = require('ember-cli-string-utils');
+const SilentError = require('silent-error');
+
+function capitalize(name) {
+  return name[0].toUpperCase() + name.slice(1);
+};
+
+function validComponentName(entityName) {
+  let parts = entityName.split('/');
+  let name = parts[parts.length - 1];
+
+  let firstCharacter = name[0];
+  if(firstCharacter !== firstCharacter.toUpperCase()) {
+    throw new SilentError("Component name must be capitalized");
+  }
+
+  return entityName;
```

Comment:
it might be nice to extract this into a micro lib like https://github.com/ember-cli/ember-cli-valid-component-name at some point.